flixel/text/FlxText.hx

```diff
@@ -35,11 +35,46 @@ import openfl.utils.AssetType;
 /**
  * Extends FlxSprite to support rendering text. Can tint, fade, rotate and scale just like a sprite. Doesn't really animate
  * though. Also does nice pixel-perfect centering on pixel fonts as long as they are only one-liners.
+ * 
+ * ## Autosizing
+ * 
+ * By default `FlxText` is fully autosize - its height and width are determined automatically
+ * by its text content. But you can control this behavior:
+ * 
+ * ```haxe
+ * // 1. fixed width and height
+ * var text = new FlxText(0, 0, 100, "Hello");
+ * text.fieldHeight = 50;
+ * // or
+ * text.fieldWidth = 100;
+ * text.fieldHeight = 50;
+ * 
+ * // 2. fixed width but auto height
+ * var text = new FlxText(0, 0, 100, "Hello");
+ * // or
+ * text.fieldWidth = 100;
+ * text.fieldHeight = -1;
+ * 
+ * // 3. autosize (by default)
+ * var text = new FlxText(0, 0, -1, "Hello");
+ * // or
+ * text.autoSize = true;
+ * text.wordWrap = false;
+ * // or
+ * text.fieldWidth = text.fieldHeight = -1;
+ * 
+ * // 4. auto width but fixed height
+ * // has no sense and works as default autosize
+ * ```
  */
 class FlxText extends FlxSprite
 {
 	/**
 	 * 2px gutter on both top and bottom
+	 * 
+	 * NOTE: Right now it is needed when new height is calculated from
+	 * `textHeight` to get a nice padding from bottom line of text
+	 * to bottom edge of text field.
```

Comment:
this is verbose and confusing, lets just do
```hx
/**
 * 2px gutter on both top and bottom when the text has dynamic height
 */

Reply:
Also. "2px on both top and bottom" is not correct, it is should be "4px at the bottom". You can clearly see it on flash target.